tools/src/main/python/dlpx/virtualization/_internal/validation_schemas/plugin_config_schema.json

```diff
@@ -54,6 +54,10 @@
       "luaName":  {
           "type": "string",
           "pattern": "^[a-z0-9_:-]+$"
+      },
+      "minimumLuaVersion": {
+          "type": "string",
+          "pattern": "^([0-9]+)\\.([0-9]+)\\.([a-zA-Z0-9_]+)$"
```

Comment:
maybe I misunderstood our conversation here but don't we have to change it to be two numbers separated by a period instead?